<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicohelpBrowser</class>
 <widget class="QWidget" name="PicohelpBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>picogen:Picohelp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="gotoMainPageButton">
        <property name="text">
         <string>Main Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gotoTutorialsButton">
        <property name="text">
         <string>Tutorials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gotoReferencesButton">
        <property name="text">
         <string>References</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>178</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Go to previous page</string>
        </property>
        <property name="icon">
         <iconset resource="../../SimplexGui/resources.qrc">
          <normaloff>:/Picohelp/resources/human_o2_media_backward.png.png</normaloff>:/Picohelp/resources/human_o2_media_backward.png.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forwardButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Go to next page</string>
        </property>
        <property name="icon">
         <iconset resource="../../SimplexGui/resources.qrc">
          <normaloff>:/Picohelp/resources/human_o2_media_forward.png</normaloff>:/Picohelp/resources/human_o2_media_forward.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="smallerButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zoom out</string>
        </property>
        <property name="icon">
         <iconset resource="../../SimplexGui/resources.qrc">
          <normaloff>:/Picohelp/resources/human_o2_zoom_minus.png</normaloff>:/Picohelp/resources/human_o2_zoom_minus.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetScalingButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Reset zoom</string>
        </property>
        <property name="icon">
         <iconset resource="../../SimplexGui/resources.qrc">
          <normaloff>:/Picohelp/resources/snow_zoom_reset.png</normaloff>:/Picohelp/resources/snow_zoom_reset.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="biggerButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zoom in</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="../../SimplexGui/resources.qrc">
          <normaloff>:/Picohelp/resources/human_o2_zoom_plus.png</normaloff>:/Picohelp/resources/human_o2_zoom_plus.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWebView" name="webView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="url">
        <url>
         <string>file:///usr/share/picogen/help-content/Main_Page.html</string>
        </url>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../SimplexGui/resources.qrc"/>
 </resources>
 <connections/>
</ui>
